src/utils/getContract.ts

```diff
@@ -0,0 +1,25 @@
+import { AddressZero } from '@ethersproject/constants'
+import { Contract } from '@ethersproject/contracts'
+import type { JsonRpcProvider, JsonRpcSigner } from '@ethersproject/providers'
+
+import { isAddress } from './addresses'
+
+// account is optional
```

Comment:
nit: all of these `// account is optional` comments aren't helpful, can you replace them with better descriptions?

Reply:
tried to think of something but i think its already expressed in the type system so just removing